src/Microsoft.Framework.PropertyHelper.Internal/PropertyHelper.cs

```diff
@@ -69,7 +75,45 @@ public static PropertyHelper[] GetProperties(object instance)
         /// </returns>
         public static PropertyHelper[] GetProperties(Type type)
         {
-            return GetProperties(type, CreateInstance, ReflectionCache);
+            return GetProperties(type, CreateInstance, PropertiesCache);
+        }
+
+        /// <summary>
+        /// <para>
+        /// Creates a caches fast property helpers that expose getters for every non-hidden get property
```

Comment:
should be "creates and caches" this is basically copied from the other overloads